SinaiStCatherine/SinaiEt002.xml

```diff
@@ -229,20 +251,20 @@ schematypens="http://relaxng.org/ns/structure/1.0"
                          <desc>The name of <persName ref="PRS13953Gabreel" role="owner"/> has been inserted probably after the production of the manuscript
                            in space left empty in the introductory supplication to <ref target="#ms_i1"/>. Later, it has been crossed out and that of 
                            <persName ref="PRS13952Heristoforos"/> has been added in the margin. In the final supplication on <locus target="#53ra"/>, the original 
-                           name has been erased anf that of <persName ref="PRS13952Heristoforos"/> been written above it.</desc>
+                           name has been erased and that of <persName ref="PRS13952Heristoforos"/> been written above it.</desc>
```

Comment:
Written above the erasure or over the erasure (since the name is erased, one cannot write above it).